static/js/apps/eval_retrieval_generation/sxs/left_right_picker.ts

```diff
@@ -0,0 +1,25 @@
+import sdbm from "sdbm";
```

Comment:
add the copyright comment

nit: maybe can just make this a general utils.ts file, so we can later add more util functions?

Reply:
Added copyright comment. Prefer not to make a general utils file based on principles in go/no-utils.

Reply:
oh, never seen go/no-utils, but good to know!!

Reply:
Ok for this PR.  But you'll find that we have utils. My feeling is, it is sometimes ok, since naming is hard and we also don't want to have a ton of files.

And there's go/no-no-utils, so I'd think of these as opinions, sort of.

Reply:
TIL about go/no-no-utils! and also the MOGHOUC: "massive organically grown hairball of unrelated code"